test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresInCompilerGeneratedCode.cs

```diff
@@ -2009,8 +2111,9 @@ static void TestAll ()
 			}
 
 			// Warnings for Reflection access to methods with Requires
-			[ExpectedWarning ("IL2026", "--TestLambdaInMethodWithRequires--")]
-			[ExpectedWarning ("IL2026", "--TestLambdaWithClosureInMethodWithRequires--")]
+
+			[ExpectedWarning ("IL2026", "--TestLambdaInMethodWithRequires--", ProducedBy = ProducedBy.Trimmer | ProducedBy.NativeAot)]
```

Comment:
Test are complaining that this used to include analyzer and analyzer warns, so there should be no change from the original. Now it would be Trimmer|Analyzer|NativeAot which is the default